sources/PPU/PPU.hpp

```diff
@@ -624,22 +114,17 @@ namespace ComSquare::PPU
 		[[nodiscard]] Vector2<bool> getBackgroundMirroring(int bgNumber) const;
 		//! @brief Render the Main and sub screen correctly
 		void renderMainAndSubScreen();
-		//! @brief Add a bg buffer to another buffer
-		template <std::size_t DEST_SIZE_X, std::size_t DEST_SIZE_Y, std::size_t SRC_SIZE_X, std::size_t SRC_SIZE_Y>
-		void add_buffer(std::array<std::array<uint32_t, DEST_SIZE_Y>, DEST_SIZE_X> &bufferDest,
-		                     const std::array<std::array<uint32_t, SRC_SIZE_Y>, SRC_SIZE_X> &bufferSrc,
-		                     const Vector2<int> &offset = {0, 0})
+		//! @brief Add a bg to the sub and/or main screen
+		template<int levelLow, int levelHigh>
```

Comment:
You should document that more, I have no idea what these template arguments are for.

Reply:
https://github.com/AnonymusRaccoon/ComSquare/pull/50#discussion_r669105545

Reply:
Documentation will be added as a comment later because doxygen doesn't supportwell template args documentation

Reply:
//! @tparam

Reply:
na it's for function parameters

Reply:
nevermind